src/webui/www/eslint.config.mjs

```diff
@@ -30,6 +30,7 @@ export default [
             "no-undef": "off",
             "no-unused-vars": "off",
             "no-var": "error",
+            "prefer-arrow-callback": "error",
```

Comment:
Wanted to note that if any of these callbacks use `this`, the behavior will change with this commit. I haven't gone through to audit usage of `this`.